docs/source/user/files.md

```diff
@@ -1,32 +1,33 @@
 
 # Working with files
 
-## Overview
+## Opening files
 
-The File Browser and File menu allow you to work with files and directories on
-your system. This includes opening, creating, deleting, renaming, downloading,
-copying and sharing files and directories.
+The file browser and File menu allow you to work with
```

Comment:
Here and elsewhere you have changed the capitalization from proper-noun-case (contrary to the developer documentation guidelines). I personally don't have a strong opinion on this, except that we should be consistent.

Reply:
I mentioned this as well.